### PR TITLE
Add librepcb.app v0.1.3

### PR DESCRIPTION
**Pull request by onesmallskipforman**
Add librepcb.app, an open-source PCB design and schematic tool.

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/librepcb.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  version '0.1.3'
+  sha256 '51fdc65e748cc99c25db90cb9babdc008a66bd4b5c5b6cbc744a7c9049cda3ca'
+
+  url "https://download.librepcb.org/releases/#{version}/librepcb-#{version}-mac-x86_64.dmg"
+  name 'LibrePCB'
+  homepage 'https://librepcb.org/'
+
+  app 'librepcb.app'
+
+  zap delete: [
```
Comment (vitorgalvao) [inline Casks/librepcb.rb:11]: If there’s only one file, make this a string instead of an array.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/librepcb.rb (hunk 1 above)
Comment (onesmallskipforman) [reply Casks/librepcb.rb]: Sounds good. I added an appcast and changed the zap stanza to a single string.

**Comment by onesmallskipforman** [comment]
@vitorgalvao I also switched the zap stanza to use trash instead of delete. Everything should be good now.

**Pull request merged**